app/controllers/books_controller.rb

```diff
@@ -5,7 +5,7 @@ class BooksController < ApplicationController
 
   # GET /books or /books.json
   def index
-    @books = Book.all
+    @books = Book.all.page(params[:page]).order(id: :asc)
```

Comment:
```suggestion
    @books = Book.order(id: :asc).page(params[:page])
```

orderを手前に持ってくることが多いかも（ただの監修ですが）

あと、orderやwhereを指定するとallは不要になります〜。